server/api/extension-hooks/routes/put_update.js

```diff
@@ -12,26 +9,6 @@ module.exports = (server) => ({
     ]
   },
   handler: (req, reply) => {
-    req.pre.auth0
-      .rules
-      .getAll()
-      .then(rules => {
-        const config = {
-          groupsInToken: true,
-          persistGroups: true
-        };
-
-        const name = 'auth0-authorization-extension';
-        const rule = _.find(rules, { name });
-
-        if (rule) {
-          return compileRule(req.storage, req.pre.auth0, config, name)
-            .then(script => req.pre.auth0.rules.update({ id: rule.id }, { name, script }));
-        }
-
-        return Promise.resolve();
-      })
-      .then(() => reply().code(204))
-      .catch((err) => reply.error(err));
+    reply().code(204);
```

Comment:
Just a comment; I'm guessing that this is a common pattern among all extensions. On extension update, this url gets requested. Keeping the endpoint live is probably fine, but it might be better to get rid of this endpoint altogether if it's not required. However it's hard to know if it'll break anything

Reply:
This is a good idea, but I am not sure which `webtask.json` is used to invoke the endpoint in the first place. While modifying the upgrade message, I noticed that the current version's message is used, not the target versions. Maybe such a thing is also the case for it. 

I prefer to leave this endpoint as noop, if possible.